revscoring/features/wikitext/datasources/parsed.py

```diff
@@ -286,6 +293,9 @@ def _extract_tag_name(tag):
 def _extract_template_name(template):
     return str(template.name)
 
```

Comment:
Flake8 will complain that there needs to be two empty lines before top-level function definitions.  Otherwise, this looks good to me.

Reply:
Oh, I totally forgot about that
Let me edit and push again?